libpod/events/filters.go

```diff
@@ -72,21 +72,15 @@ func generateEventUntilOption(timeUntil time.Time) func(e *Event) bool {
 	}
 }
 
-func parseFilter(filter string) (string, string, error) {
-	filterSplit := strings.Split(filter, "=")
-	if len(filterSplit) != 2 {
-		return "", "", errors.Errorf("%s is an invalid filter", filter)
-	}
-	return filterSplit[0], filterSplit[1], nil
+func parseFilter(filter string) (string, string) {
```

Comment:
This still needs to return (string, string, error). You need to check for length == 2 (otherwise we segfault on filterSplit[1] if the user passes an invalid filter)